app/views/payments/ChargeTypeDetailsView.scala.html

```diff
@@ -0,0 +1,130 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.viewModels.WhatYouOweChargeModel
+@import views.templates.formatters.dates.DisplayDateRangeHelper._
+
+@this(mainTemplate: MainTemplate,
+      govukBreadcrumbs: GovukBreadcrumbs,
+      govukSummaryList: GovukSummaryList,
+      govukButton: GovukButton,
+      govukTag: GovukTag)
+
+@(model: WhatYouOweChargeModel, serviceInfoContent: Html)(
+  implicit request: Request[_], messages: Messages, appConfig: config.AppConfig, user: User)
+
+@breadcrumbs = {
+    @govukBreadcrumbs(Breadcrumbs(
+        items = Seq(
+            BreadcrumbsItem(
+                content = Text(messages("breadcrumbs.bta")),
+                href = Some(appConfig.btaHomeUrl)
+            ),
+            BreadcrumbsItem(
+                content = Text(messages("vatDetails.title")),
+                href = Some(controllers.routes.VatDetailsController.details.url)
+            ),
+            BreadcrumbsItem(
+                content = Text(messages("openPayments.title")),
+            )
+        )
+    ))
+}
+
+@dueDateHtml = {
+    @model.dueDate
+}
+
+@chargeDueHtml = {
+    @model.originalAmount
+}
+
+@clearedAmountHtml = {
+    @model.clearedAmount.getOrElse(0)
+}
+
+@outstandingAmountHtml = {
+    @model.outstandingAmount
+}
+
+@mainTemplate(
+    title = model.chargeTitle,
+    appConfig = appConfig,
+    serviceInfoContent = serviceInfoContent,
+    user = Some(user),
+    navLinkContent = Some(breadcrumbs)
```

Comment:
I think we neglected the agent requirements in refinement & subtasking but agents should just have a Back link to WYO, rather than breadcrumbs.